packages/neuron-wallet/src/controllers/export-debug.ts

```diff
@@ -95,30 +97,31 @@ export default class ExportDebugController {
         release,
         vcredist,
       },
+      wallets: wallets.map(v => ({
+        id: v.id,
+        name: v.name,
```

Comment:
`Name` of a wallet is kind of private and it's better not to be exposed in public, and `id` is sensitive too, how about masking some chars of the id in the log

Reply:
I will remove the name field. Id is a random string created by `uuid`, I don't know whether id is sensitive

Reply:
I have removed the `walletId` in `hd_public_key_info.csv` and used `index` to recognize it as one wallet, and also match the start block number that the user sets.